test/CMakeLists.txt

```diff
@@ -95,6 +95,17 @@ ADD_TEST( t_${test_name} "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh
     SET_TESTS_PROPERTIES( t_${test_name} PROPERTIES FAIL_REGULAR_EXPRESSION  " Exception; EXCEPTION;ERROR;Error" TIMEOUT 700)
 endif()
 
+#--------------------------------------------------
+# test for IDEA o1 v03, with DRC
+if(DCH_INFO_H_EXIST)
+SET( test_name "test_IDEA_with_DRC_o1_v03" )
+ADD_TEST( t_${test_name} "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh"
+	"ddsim --compactFile=/home/alvarotd/work/update_main_detectors_after_DRC/k4geo/test/compact/IDEA_withDRC_o1_v03.xml --steeringFile=/home/alvarotd/work/update_main_detectors_after_DRC/k4geo/test/../example/SteeringFile_IDEA_o1_v03.py --gun.direction '(3,4,0)'
```

Comment:
```suggestion
	"ddsim --compactFile=${CMAKE_CURRENT_SOURCE_DIR}/I_dont_event_know_/k4geo/test/compact/IDEA_withDRC_o1_v03.xml --steeringFile=${CMAKE_CURRENT_SOURCE_DIR}/I_dont_event_know_/../example/SteeringFile_IDEA_o1_v03.py --gun.direction '(3,4,0)'
```

Reply:
that was a desesperate trial using absolute paths, but keeps failing in the same way...